.travis.yml

```diff
@@ -23,7 +23,7 @@ sudo: required
 
 script:
   - export GO111MODULE=on
-  - make lint unit build mod-check
+  - make lint unit build mod-check rpc-js-compile
```

Comment:
nice, I was def going to manage to break the build somehow in future so thank you 🙏